addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -151,7 +151,7 @@ export const CLIPBOARD_WHITELISTS = {
         /^fa/,
     ],
     attributes: ['class', 'href', 'src', 'target'],
-    styledTags: ['SPAN', 'B', 'STRONG', 'I', 'S', 'U', 'FONT'],
+    styledTags: ['SPAN', 'B', 'STRONG', 'I', 'S', 'U', 'FONT', 'TD'],
```

Comment:
This allows cells to have style

Reply:
@dmo-odoo (in case you have to be tagged when PR is still in draft)

Reply:
@dmo-odoo I'll remove this change and put the branch in testing. Unless you have something against it, I think this is the most controversial line of code in this PR and removing it only means we don't support styling on cells but that makes more sense in v15. I'd love to add it in the v16 forward-port if you think that's fine.
The rest of the PR seems OK to me although I'm not sure you'll agree with my `styleMap` below.